backend/app.ts

```diff
@@ -13,6 +13,7 @@ import teamRoutes from './routes/teamRoutes';
 import teamSetRoutes from './routes/teamSetRoutes';
 import userRoutes from './routes/userRoutes';
 import { connectToDatabase } from './utils/database';
+import setupCodeAnalysisJob from 'jobs/codeAnalysisJob';
```

Comment:
I think to be safe, change this to `./jobs/codeAnalysisJob`, similar to the other jobs import. I have seen build errors in staging docker container when using different kinds of imports.

Same for other files, follow the same import format to be safe. Like for `backend/jobs/codeAnalysisJob.ts`, check `githubJob` and jiraJob` and use the same format for imports.